packages/algorand-js/src/internal/types.ts

```diff
@@ -0,0 +1,10 @@
+import {
+  Add, Arg_0, Arg_1, Arg_2,
+  Arg_3, Div, Len, Mul, Sub
+} from "../internal/interpreter/opcode-list";
+
+export type Operator = Len | Add | Sub |
+Mul | Div | Arg_0 | Arg_1 | Arg_2 | Arg_3;
+
+export type AppArgs = Array<string | number>;
+export type StackElem = string | bigint;
```

Comment:
I've verified the opcodes.
It seams that the stack values are either `[]byte` or `uint64`. Hence we should change `string` to `Uint8Array`

Reply:
Maybe we can create a shortcut type, and use it in the interpreter:
```
type TEALStack = IStack<BigInt | Uint8Array>
```

Reply:
Okay, makes sense.